estuary/utils/story.py

```diff
@@ -0,0 +1,391 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from __future__ import unicode_literals
+
+from neomodel import db
+
+from estuary.error import ValidationError
+from estuary.models import story_flow_list, module_story_flow_list
+
+
+class StoryUtils(object):
+    """A class containing utility methods to create a story for an artifact."""
+
+    def __init__(self, module_flow=False):
+        """
+        Instantiate variables required to get story of a node.
+
+        :kwarg bool module_flow: specifies if the story flow with modules is to
+        be taken for this artifact
+        """
+        self.story_order_list = module_story_flow_list
```

Comment:
Instead of a parameter of `module_flow` being passed in, how about converting this specific class to a base class and then you can have sub-classes for each story type. So you'd have a `ModuleStoryManager` class and a `ContainerStoryManager`. Each sub-class would define the story flow variable. In this event, the `ModuleStoryManger` class would likely inherit from `ContainerStoryManager` and then call the parent's `story_flow` method and update the return value.